src/Tizen.NUI/src/public/Accessibility/IAtspiTable.cs

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright(c) 2022 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using Tizen.NUI.BaseComponents;
+
+namespace Tizen.NUI.Accessibility
+{
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public interface IAtspiTable
+    {
```

Comment:
As you know, we need to add descriptions here even though all functions' explanation is same as DALi one.
The reason is that this file is public one.

However, the description of all Accessibility Interface files is currently not written. So, let's add them together in the other PR later 😏

Reply:
Correct, none of the other Accessibility interfaces have got any documentation in this repository… I agree, this should be a separate PR. :smile: